frontend/packages/kubevirt-plugin/integration-tests/tests/vm.environment.scenario.ts

```diff
@@ -0,0 +1,133 @@
+import { getSecret, getConfigMap, getServiceAccount } from './utils/mocks';
+import { createResource, deleteResource, click } from '@console/shared/src/test-utils/utils';
+import { browser, ExpectedConditions as until, Key, element, by } from 'protractor';
+import { createExampleVMViaYAML } from './utils/utils';
+import { testName, appHost } from '@console/internal-integration-tests/protractor.conf';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import * as vmEnv from '../views/vm.environment.view';
+import { addVariableFrom } from '@console/internal-integration-tests/views/environment.view';
+import {
+  PAGE_LOAD_TIMEOUT_SECS,
+  KUBEVIRT_SCRIPTS_PATH,
+  VM_BOOTUP_TIMEOUT_SECS,
+  VM_ACTION,
+} from './utils/consts';
+import { execSync } from 'child_process';
+import { VirtualMachine } from './models/virtualMachine';
+
+const expecScriptPath = `${KUBEVIRT_SCRIPTS_PATH}/expect-vm-env-readable.sh`;
+const vmName = 'vm-example';
+const configmapName = 'configmap-mock';
+const secretName = 'secret-mock';
+const serviceAccountName = 'service-account-mock';
+
+describe('Test VM enviromnet tab', () => {
+  const secret = getSecret(testName, secretName);
+  const configmap = getConfigMap(testName, configmapName);
+  const serviceAccount = getServiceAccount(testName, serviceAccountName);
+  let vm: VirtualMachine;
+
+  beforeAll(async () => {
+    createResource(secret);
+    createResource(configmap);
+    createResource(serviceAccount);
+    vm = await createExampleVMViaYAML(true);
+  });
+
+  afterAll(() => {
+    deleteResource(secret);
+    deleteResource(configmap);
+    deleteResource(serviceAccount);
+  });
+
+  beforeEach(async () => {
+    await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines/${vmName}/environment`);
```

Comment:
Please, add Environment tab to `packages/kubevirt-plugin/integration-tests/tests/utils/consts.ts`
and then you can use `await vm.navigateToTab(TAB.Environment)`